src/MarkdownTextInput.web.tsx

```diff
@@ -520,6 +520,10 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
       divRef.current.focus();
     }, []);
 
+    const startComposition = () => {
+      compositionRef.current = true;
+    };
```

Comment:
Can we memoize this callback so it's not regenerated on each render?
```suggestion
    const startComposition = useCallback(() => {
      compositionRef.current = true;
    }, []);
```